pkg/types/default.go

```diff
@@ -9,6 +9,9 @@ func DefaultPlatformConfig() PlatformConfig {
 		CertManager: CertManager{
 			Version: "v1.0.3",
 		},
+		IstioOperator: IstioOperator{
```

Comment:
I don't think this should be installed by default

Reply:
Any suggestions about which test to use?

Reply:
Use platform